stories/views.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.shortcuts import render
+from rest_framework import generics
+
+from .serializers import ResponsesSerializer, UserSerializer
+from .models import Responses
+
+class ResponsesList(generics.ListCreateAPIView):
+    '''
+    This class defines creation property of the api.
+    '''
+    queryset = Responses.objects.all()
+    serializer_class = ResponsesSerializer
+
+    def create_story(self, serializer):
```

Comment:
Do you have to call this method? Doesn't the super class automatically run this for you